source/expressions.tex

```diff
@@ -3277,29 +3277,22 @@
 ``\cvqual{cq12} \cvqual{vq12} \tcode{T}''.
 
 \item If \tcode{E2} is a (possibly overloaded) member function, function
-overload resolution\iref{over.match} is used to determine whether
-\tcode{E1.E2} refers to a static or a non-static member function.
+overload resolution\iref{over.match} is used to determine to which
+function \tcode{E1.E2} refers.
+If \tcode{E2} refers to a member function of type \tcode{T},
+\tcode{E1.E2} has type \tcode{T} and refers to that member function.
 
 \begin{itemize}
-\item If it refers to a static member function and the type of
-\tcode{E2} is ``function of parameter-type-list returning \tcode{T}'',
-then \tcode{E1.E2} is an lvalue; the expression designates the static
-member function. The type of \tcode{E1.E2} is the same type as that of
-\tcode{E2}, namely ``function of parameter-type-list returning
-\tcode{T}''.
+\item If \tcode{E1.E2} refers to a static member function,
```

Comment:
I think "If `E2` refers to a static member function" would be a little simpler.

Reply:
Fixed.